src/main/java/io/temporal/samples/ssl/SslEnabledWorker.java

```diff
@@ -0,0 +1,74 @@
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.samples.ssl;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowClientOptions;
+import io.temporal.serviceclient.SimpleSslContextBuilder;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+import io.temporal.serviceclient.WorkflowServiceStubsOptions;
+import io.temporal.worker.Worker;
+import io.temporal.worker.WorkerFactory;
+import java.io.FileInputStream;
+import java.io.InputStream;
+
+public class SslEnabledWorker {
+
+  static final String TASK_QUEUE = "MyTaskQueue";
+
+  public static void main(String[] args) throws Exception {
+    // Load your client certificate, which should look like:
+    // -----BEGIN CERTIFICATE-----
+    // ...
+    // -----END CERTIFICATE-----
+    InputStream clientCert = new FileInputStream(System.getenv("TEMPORAL_CLIENT_CERT"));
+    // PKCS8 client key, which should look like:
+    // -----BEGIN PRIVATE KEY-----
+    // ...
+    // -----END PRIVATE KEY-----
+    InputStream clientKey = new FileInputStream(System.getenv("TEMPORAL_CLIENT_KEY"));
+    // For temporal cloud this would likely be ${namespace}.tmprl.cloud:7233
+    String targetEndpoint = System.getenv("TEMPORAL_ENDPOINT");
+    // Your registered namespace.
+    String namespace = System.getenv("TEMPORAL_NAMESPACE");
+    // Create SSL enabled client by passing SslContext, created by SimpleSslContextBuilder.
+    WorkflowServiceStubs service =
+        WorkflowServiceStubs.newInstance(
+            WorkflowServiceStubsOptions.newBuilder()
+                .setSslContext(SimpleSslContextBuilder.newBuilder(clientCert, clientKey).build())
+                .setTarget(targetEndpoint)
+                .build());
+
+    // Now setup and start workflow worker, which uses SSL enabled gRPC service to communicate with
+    // backend.
+    // client that can be used to start and signal workflows.
+    WorkflowClient client =
+        WorkflowClient.newInstance(
+            service, WorkflowClientOptions.newBuilder().setNamespace(namespace).build());
+    // worker factory that can be used to create workers for specific task queues
+    WorkerFactory factory = WorkerFactory.newInstance(client);
+    // Worker that listens on a task queue and hosts both workflow and activity implementations.
+    Worker worker = factory.newWorker(TASK_QUEUE);
+    // TODO now register your workflow types and activity implementations.
```

Comment:
This TODO, is it meant for the person reading the sample?

If so maybe remove the TODO and say "Now, here is where you would register... as in the below two lines"

Reply:
I wanted to make it more obvious that it's a required part, and didn't want to copy-paste some other sample, as it's not focused on the specific behavior. Having TODO would highlight the line in most people's IDE, making it more clear.

Reply:
Ah, sure that makes sense. Might keep the TODO and the wording upgrade so it's a bit more clear it's a call to action.